staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -2510,6 +2510,8 @@ type VirtualMachineInstanceGuestAgentInfo struct {
 	// It will be set to "frozen" if the request was made, or unset otherwise.
 	// This does not reflect the actual state of the guest filesystem.
 	FSFreezeStatus string `json:"fsFreezeStatus,omitempty"`
+	// Load contains the system load averages (1M, 5M, 15M) from the guest agent
+	Load VirtualMachineInstanceGuestOSLoad `json:"load,omitempty"`
```

Comment:
How often do we expect this to be updated? What's the added load on the API server?
Does this fit in status, or do we only want to expose this via metrics?